### PR TITLE
eqc documentation matching template (version 3)

### PR DESCRIPTION
**Pull request by julien2512**
Here is the third version, matching the template proposal.

As I got a better understanding of proj4 implementation, I think there might be a mistake in eqc implementation.

In the description of wikipedia, nor wolfram, there is only one parameter \phi_ts. Evenden do not use any.
But in proj4, we got two : \phi_ts and \phi_0.

That's why I tryed to document it on this pull request.
Although, I can't find how this value is valuated in proj4, and I find no documentation (wikipedia, evenden, wolfram, epsg) talking about a parameter phi_0 different than phi_ts.

May be phi_0 value is simply always 0 in proj4, and just leave in the code as a reminder ?

### CODE REVIEW

File: docs/source/projections/eqc.rst
Comment (QuLogic) [inline docs/source/projections/eqc.rst]: simplest

File: docs/source/projections/eqc.rst
Comment (QuLogic) [inline docs/source/projections/eqc.rst]: Carree, or even more correct, Carrée, especially because that's used later.

File: docs/source/projections/eqc.rst
Comment (QuLogic) [inline docs/source/projections/eqc.rst]: This is the "White Bullet" symbol, not the degree sign, cf. ◦ vs. °.

File: docs/source/projections/eqc.rst
Comment (QuLogic) [inline docs/source/projections/eqc.rst]: The "apostrophe" should be a "prime", cf. ' vs. ′ (though I can't really see a difference with my default font here, so it's rather minor, and probably not a big deal.)

File: docs/source/projections/eqc.rst
Comment (QuLogic) [inline docs/source/projections/eqc.rst]: Carree or Carrée

**Review by QuLogic — CHANGES_REQUESTED** [review]
Mostly minor tweaks to stuff.

File: docs/source/projections/eqc.rst
Comment (kbevers) [inline docs/source/projections/eqc.rst:74]: This section would be more clear if some of the variables are defined here. Like phi_ts is the latitude of true scale and can be set with +lat_ts.

**Review by kbevers — CHANGES_REQUESTED** [review]
Good work @julien2512. Thanks for doing this.

I've pointed out a few things I'd like to see adressed before I merge this. On top of this I think that @QuLogic has some valid points as well. 

Regarding the phi_0 variable. It is the latitude of origin and can be set with +lat_0.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: done.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: Alright.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: Really ? I hope parsers will recognise both.
done.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: It was from the old doc !
done.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: Even better !
done.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: There is a point here. The plate carrée is just one case of equidistant cylindrical, isn't it ? So why using it on the title ...

done.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst]: done

**Comment by julien2512** [comment]
@kbevers 

> Regarding the phi_0 variable. It is the latitude of origin and can be set with +lat_0.

Can you help me to know where the lat_0 is taking a value in proj4 ? It can make me ask many less questions !

**Comment by kbevers** [comment]
> Can you help me to know where the lat_0 is taking a value in proj4 ? It can make me ask many less questions !

Sure. It's done when calling pj_init. So when you initialize a projection.

**Comment by julien2512** [comment]
Thank you, I found :
`PIN->phi0 = pj_param(ctx, start, "rlat_0").f;`

**Comment by julien2512** [comment]
@QuLogic thank you for your help.

File: docs/source/projections/eqc.rst
Comment (julien2512) [reply docs/source/projections/eqc.rst:74]: I tryed something with the parameters.
You may have a look on it ?
I feel it is important to evaluate the quality before stepping to an other projection doc.

File: docs/source/projections/eqc.rst
Comment (kbevers) [inline docs/source/projections/eqc.rst:75]: I think there's a backslash missing here.  Same for the line below.

**Comment by julien2512** [comment]
This is working for me without backslashes. Is there a better way to format
these lines ?

Le 16 sept. 2016 23:27, "Kristian Evers" notifications@github.com a
écrit :

**Comment by kbevers** [comment]
Really? Do you build the docs with sphinx?

**Comment by julien2512** [comment]
Of course ! 
Eventhough, I am open to your proposal.

**Comment by kbevers** [comment]
Great. I'll merge your PR once you change it.

**Comment by kbevers** [comment]
Thanks for writing this, Julien. It is very much appreciated!

**Comment by julien2512** [comment]
Let's start other ones !

**Comment by julien2512** [comment]
It will be Cassini.

**Comment by Andre-J** [comment]
There is a bug in the example: 
Example using Plate Carrée projection with true scale at latitude 30° and central meridian 90°W:
$ echo -88 30 | proj +proj=eqc +lat_ts=30 +lat_0=90w

Central meridian should be lon_0, which is not included in the formulas, but changing values for lon_0 different from zero resolves to different results. Seems that Proj can do more than Snyder wrote in his formulas. Maybe this should be pointed out on the manpage.

**Comment by kbevers** [comment]
Good catch. Indeed it should be `+lon_0`. Fixed in https://github.com/OSGeo/proj.4/commit/89af2753651f8c21eae9df631a71c0a3b03bcc3b

> Seems that Proj can do more than Snyder wrote in his formulas. Maybe this should be pointed out on the manpage.

The `+lon_0` adjust the longitude before it reaches the projection fwd/inv functions, as seen here: https://github.com/OSGeo/proj.4/blob/84c2fed374da46ba0ca3cae1125350b1aa3f73fd/src/pj_fwd.c#L86

I agree that this should be documented better. It's on my list for the next large overhaul of the docs.

**Pull request merged**
